Documentation/policy.rst

```diff
@@ -669,10 +669,10 @@ endpoint with the label ``id.http`` on port 80:
     Tracing From: [container:id.curl] => To: [container:id.httpd] Ports: [80/any]
     * Rule 2 {"matchLabels":{"any:id.httpd":""}}: match
         Allows from labels {"matchLabels":{"any:id.curl":""}}
-    +     Found all required labels
+          Found all required labels
+            Restricted to specific L4 destinations; skip during Label stage
```

Comment:
Along with this change to the tracing output, I think we should add documentation to this file explaining L3-dependent L4 policy as well. I don't mind this being done after this patch is merged in the interest of having you not have to rebase again :)

Reply:
I'll take note to follow up on this. I think that the way that the documentation is written right now, it already implies the behaviour specified with this l3-dependent l4 work. I just don't think that we have any examples using multiple ingress rules, so we could highlight the behaviour difference. This could help people who end up in a situation where they're allowing L3 and allowing L4, but actually intended to only specify l3-dependent l4 policy.

Reply:
Great, sounds good! Please create a doc ticket for that so we have it tracked.

Reply:
Done: https://github.com/cilium/cilium/issues/1685